src/site_apportionment/site_apportionment.py

```diff
@@ -217,13 +218,15 @@ def create_category_df(
         groupby_cols (List[str]): List of columns to group by.
         code_cols (List[str]): List of code columns.
         site_cols (List[str]): List of site column.
+        geo_cols (List[str]): List of geographic columns from the config.
```

Comment:
geo cols doesn't include itl - will that need adding so it appears in GBSAS correctly?  Not sure if that needs to be kept separate so it doesn't impact the intram itl output

Reply:
good point, thanks